src/factories/createRoarrInitialGlobalState.ts

```diff
@@ -13,14 +13,16 @@ export default (currentState: any): RoarrGlobalState => {
     versions.sort(cmp);
   }
 
-  const currentIsLatestVersion = !versions.length || cmp(pkg.version, versions[versions.length - 1]) === 1;
+  const currentIsLatestVersion = versions.length < 2 || cmp(pkg.version, versions[versions.length - 1]) === 1;
```

Comment:
I'm not sure what you want to express here

the original code is checking for no presence in the array, while this version is checking for that but also if one item is present